### PR TITLE
Added packshots for articles

### PR DESCRIPTION
**Pull request by ivan-wolf**

### CODE REVIEW

File: src/swagger.yml [hunk 1]
```diff
@@ -2475,6 +2487,8 @@ definitions:
         $ref: '#/definitions/AssignedArticleGroup'
       eco_labels:
         $ref: '#/definitions/EcoLabels'
+      packshots:
```
Comment (curzio-della-santa) [inline src/swagger.yml]: packshots does not belongs in Article definition, should be removed

File: examples/articles.json [hunk 2]
```diff
@@ -126,6 +126,14 @@
                     "name": "Bio"
                 }
             ],
+            "packshots": [
```
Comment (curzio-della-santa) [inline examples/articles.json]: packshots does not belongs in articles, we do not provide such information

File: src/swagger.yml [hunk 3]
```diff
@@ -713,6 +713,12 @@ paths:
                     id: "66d94617-bbd7-477e-ae66-183deb67ba5b"
                     code: "bio"
                     name: "Bio"
+                packshots:
+                  -
```
Comment (curzio-della-santa) [inline src/swagger.yml]: here has to be removed too

**Comment by ivan-wolf** [comment]
Merge after it has been implemented.

**Pull request merged**